primap2/pm2io/_data_reading.py

```diff
@@ -1506,9 +1516,9 @@ def harmonize_units(
         data_this_entity = data.loc[data[entity_col] == entity]
         units_this_entity = data_this_entity[unit_col].unique()
 
-        if len(units_this_entity) > 1 or gwp_to_use:
+        if len(units_this_entity) > 1:
```

Comment:
What happens if `gwp_to_use` is set, but `len(units_this_entity) = 1`? I.e. I have a GWP entity in a single unit. Previously, I would get unit conversion, now not any more. Is that intended? I see that we don't really need unit conversion in that case, as things are harmonious already, just checking.